src/components/hooks/useLinks.ts

```diff
@@ -6,7 +6,7 @@ import { useStorybookState } from "@storybook/api";
 import { getZeplinLinksFromConnectedComponents } from "../../utils/api";
 
 const getProjectIdFromProjectLink = (link: string): string | null => {
-    if (link.startsWith(`${ZEPLIN_APP_BASE}/project?`)) {
+    if (link.startsWith(`${ZEPLIN_APP_BASE}//project?`)) {
```

Comment:
Why 2 `/` chars?

Reply:
Because the app url contains 2 `/` chars (e.g. `zpl://project?pid=...`) and `ZEPLIN_APP_BASE` is `zpl:`. I admit that changing the app base to `zpl://` would be better; however, this change also affects the `zeplin-inspector` package. To keep the changeset small, I did it this way.